### ISSUE TITLE
initialize with last hyperdrive hash opened

### ISSUE THREAD

**Issue by clkao** [karissa/dat-explorer#42]
When no hash is given, instead of always creating a new one, use the previous one if user has visited before.

**Comment by mafintosh** [comment]
👍 x 💯

**Comment by okdistribute** [comment]
could we have a list of the previous links they've visited in the local storage/indexed db until we have user accounts/repos?

**Comment by juliangruber** [comment]
> could we have a list of the previous links they've visited in the local storage/indexed db until we have user accounts/repos?

definitely! we could start with the full history in local storage, then add the feature to reopen the last one automatically, then add ui for browsing all (unless accounts are already live by then)
